src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -43,6 +43,19 @@
     #endif
 #endif
 
+#if defined(Regulatory_Domain_EU_CE_LBT_2400)
+    #undef MinPower
+    #define MinPower PWR_10mW
```

Comment:
Some modules can't do 10mW so this will cause problems.

Reply:
How about merging the two EU_CE domains? All modules have power levels <=100mW. Only modules that can do 10mW would be able to run with LBT disabled.

Reply:
> Only modules that can do 10mW would be able to run with LBT disabled.
Wouldn't this decision be up to the manufacturer, there is no point in running with these modes if the equipment is not CE-marked, its out of spec anyway without sticker. And with sticker, the manufacturer decides one or the other, can't mix and match them either I think. Unless tested in both modes, which adds test-cost etc.

Reply:
But I guess for the rest of us, it is also nice if all of this makes logical sense and we can mix and match without ending up with some weird config because of limitations like this.

Reply:
My point was more about the separate LBT and non-LBT CE regulatory domains. Assuming LBT is implemented correctly, I don't see a reason why someone would flash the 10mW limited non-LBT CE reg domain, or why a manufacturer would be interested in certifying their hardware with a 10mW power limit. If you merge them into a single CE reg domain, then there is no need to redefine `minPower` because all modules can transmit at or below 100mW. The only thing to add is a check that disables LBT when configured power is 10mW. Modules that cannot do 10mW cannot select this power level so for them LBT will always be enabled. If a manufacturer really wants to release a module that only does 10mW then they can put that in the target definition and the results would be the same as with the current non-LBT CE reg domain.

Reply:
ok, so if 10mW is configured, and no dynamic power, we just disable LBT. I wonder if that is allowed if the product was only certified with LBT. If LBT is enabled, is also must operate for 10mW.

Reply:
> ok, so if 10mW is configured, and no dynamic power, we just disable LBT.

That's how I understood the rules. When manually setting power to 10mW no LBT is required. When a higher power level is configured but dynamic power is enabled and power drops down to 10mW then LBT must still be used. I think @JyeSmith was going to ask around to confirm this but to me, it makes a lot of sense.

Reply:
yeah, when you explain it like that, I also think it makes a lot of sense. I'm currently fixing the other issue we discussed on discord with potentially wrong RSSI for RX because of too little waiting time there.

Reply:
@avsaase now RSSI for RX should be more accurate!

Reply:
@avsaase @pkendall64  I committed another attempt at this logic, but definitely want another pair of eyes at this.

The assumption is that modules either support 10mW or not, but all support 100mW or less. 
* if EU_CE non-LBT mode is defined and module supports 10mW, lock maxPower also to 10mW and stick to non-LBT
* If EU_CE non-LBT mode is defined, but minPower > 10mW for this module, it will switch to LBT-mode define. 
* If LBT-mode defined, redefine maxPower to 100mW

opinions?

Reply:
I think I'd prefer the middle case to fail the build rather than silently switching to LBT mode.

Reply:
Is there a reason to keep the two separate domains?

Reply:
> Is there a reason to keep the two separate domains?

Probably not. Just EU_CE_LBT, and if it's 10mW max then no LBT, otherwise LBT and maxPower 100mW. Job done.

I have to say, you guys have done a great job taking this on 👍

Reply:
I would call it EU_CE without the LBT part because LBT is just a requirement for CE.

Reply:
Makes perfect sense 👍

Reply:
@pkendall64 when a hw manufacturer takes this through compliance testing, they have to choose one of the three options here:
 
![image](https://user-images.githubusercontent.com/2505665/149544572-a3d291fc-593b-4ee5-8b68-08b962c2c5df.png)

Should we just assume/require everyone to select 3rd option, capable of both LBT and non-LBT modes? I understand the above proposal as such?

Reply:
Something like this:
`#if defined(Regulatory_Domain_EU_CE_2400)`
`    #if MaxPower > PWR_100mW`
`        #error "Regulatory_Domain_EU_CE_2400 only valid up to 100mW"`
`    #endif`
`    #if MinPower > PWR_10mW`
`        #define LBT_ACTIVE 1`
`    #endif`
`#endif`

and only keep Regulatory_Domain_EU_CE_2400 in user_defines, and implicitly require all ELRS CE compliant hardware to be compliance tested for both modes of operation. I think that makes sense, this is not something that adds a lot to test-cost or anything

Reply:
I've realized these macro comparisons won't work, since the PWR_100mW for example, are enums, so comparison like this won't work for the pre-processor. Any other clever ideas?

Reply:
Latest commit is an attempt at solving this, but gut-feeling says there could be prettier solutions...

Reply:
> @pkendall64 when a hw manufacturer takes this through compliance testing, they have to choose one of the three options here:
> 
> Should we just assume/require everyone to select 3rd option, capable of both LBT and non-LBT modes? I understand the above proposal as such?

Yes I would think that if we have 10mW no dynamic does not do LBT and dynamic or higher than 10mW does LBT then they would have to choose option 3.